apps/gp2-frontend/package.json

```diff
@@ -14,41 +14,39 @@
     "release:frontend": "node ./scripts/sentry-release.js"
   },
   "dependencies": {
+    "@asap-hub/algolia": "workspace:*",
     "@asap-hub/auth": "workspace:*",
+    "@asap-hub/fixtures": "workspace:*",
     "@asap-hub/flags": "workspace:*",
     "@asap-hub/frontend-utils": "workspace:*",
     "@asap-hub/gp2-components": "workspace:*",
+    "@asap-hub/model": "workspace:*",
```

Comment:
We have dictionaries and stuff we use in prod code so its a dependency not a dev dependency. Not sure how i feel about this but ESLint called us out.